.circleci/config.yml

```diff
@@ -1414,3 +1449,16 @@ workflows:
       #     report: true
       #     requires:
       #       - Build app - Windows (stage)
+
+weekly:
+  triggers:
+    - schedule:
+        cron: '0 0 * * 1'
+        filters:
+          branches:
+            only:
+              - main
+  jobs:
+    # Process all licenses
+    - licenses-check:
```

Comment:
also if we have such checks on weekly basis, why do we need it for feature/main branches?

Reply:
It was for test, removed